.circleci/config.yml

```diff
@@ -231,3 +231,39 @@ jobs:
                 --ignore tests/integration_tests/test_catalyst.py
 
       - run: *tests-mn
+
+  tests-rdbstorage:
+    docker:
+      - image: circleci/python:3.7
+      - image: circleci/redis:5.0.7
+      - image: circleci/mysql:5.7
+        environment:
+          MYSQL_DATABASE: optunatest
+          MYSQL_USER: user
+          MYSQL_PASSWORD: test
+      - image: circleci/postgres:10.1-alpine
+        environment:
+          POSTGRES_USER: user
+          POSTGRES_DB: optunatest
+          POSTGRES_PASSWORD: test
+    steps:
+      - checkout
+
+      - run: *checkout-merge-master
+
+      - run: *install
+
+      - run: &tests-rdb
+          name: tests-rdb
+          command: |
+            . venv/bin/activate
+            # Install RDB bindings for storage tests.
+            pip install --progress-bar off PyMySQL cryptography psycopg2-binary
+
+            export TEST_DB_URL=mysql+pymysql://user:test@127.0.0.1/optunatest
+            pytest tests/storages_tests/rdb_tests/test_with_server.py
+
+            export TEST_DB_URL=postgres+psycopg2://user:test@127.0.0.1/optunatest
+            pytest tests/storages_tests/rdb_tests/test_with_server.py
```

Comment:
A detail but how about using separate `run`s for different DB backends since they're quite independent?

Reply:
Sounds good! I divided the step in https://github.com/optuna/optuna/pull/1497/commits/15356075ed94d2fccd2c74ba61cbb9395016e09a.

Reply:
Thanks!